package.json

```diff
@@ -2,25 +2,46 @@
   "name": "single-spa",
   "version": "5.9.3",
   "description": "The router for easy microfrontends",
-  "main": "lib/umd/single-spa.min.js",
+  "main": "lib/umd/single-spa.min.cjs",
   "types": "typings/single-spa.d.ts",
+  "type": "module",
+  "exports": {
```

Comment:
I tested these in Node 10, Node 16 (esm importing esm, cjs importing esm, and cjs importing cjs, both for prod and dev via `node --conditions=development`). I also tested it in webpack 4, webpack 5, and rollup (with `@rollup/plugin-node-resolve`). Seems to work.